src/Flex.php

```diff
@@ -1103,4 +1106,27 @@ private function doesRecipeConflict(array $recipeData, OperationInterface $opera
 
         return false;
     }
+
+    private function symfonyRequireVersion(Composer $composer): string
+    {
+        $package = $composer->getPackage();
+        $version = getenv('SYMFONY_REQUIRE') ?: '';
+
+        if (empty($version)) {
+            $version = $package->getExtra()['symfony']['require'] ?? '';
+        }
+
+        if (empty($version)) {
+            $require = $package->getRequires();
+            $link = $require['symfony/framework-bundle'] ?? null;
```

Comment:
This means that having framework-bundle in your deps will restrict the possible symfony packages.
This is not how it behaves right now, nor what we want.